heron/stmgr/tests/cpp/server/stmgr_unittest.cpp

```diff
@@ -1445,11 +1440,6 @@ TEST(StMgr, test_tmaster_restart_on_same_address) {
 
   int num_msgs_sent_by_spout_instance = 100 * 1000 * 1000;  // 100M
```

Comment:
This is not my code, it shows up as diff context... Also, the 100M there might mean 100 million rather than 100 megabytes.